subtitles/ZAP Chat/README.md

```diff
@@ -0,0 +1,35 @@
+# Generate subtitles for ZAP Chat using whisper
+
+Here is a short guide on how the subtitles were created using OpenAI's Whisper.
+A <b>Nvidia graphics card with Cuda cores is required</b>. The more VRAM the better, as larger models can then be used if necessary.
```

Comment:
**bold** is `**bold**` 😁 
Also one below..

Reply:
nobody has seen